Docs missing _themes folder

I cloned the repo, installed sphinx, and tried to "make html" in the docs folder, however I got the following error:

Theme error:
no theme named 'sphinx_rtd_theme' found (missing theme.conf?)
make: *** [html] Error 1

After looking in conf.py, I noticed that the html_theme_path was specified as '_themes', but there is no _themes folder in the docs directory.  Was the themes folder accidentally ignored..?

--- a/docs/REAME.md
+++ b/docs/REAME.md
@@ -3,6 +3,18 @@ Documentation
 
 Project documentation is built using [Sphinx docs](http://sphinx-doc.org/), which uses [ReST](http://docutils.sf.net/rst.html) for markup.  This allows the docs to cover a vast amount of topics without using a thousand-line README file.
 
-Sphinx docs is pip-installable via `pip install sphinx`.  Once installed, open a command line in the docs folder and run `make html`; the output files will be placed in the `_build/html/` directory, and can be browsed (locally) with any browser.
+Sphinx docs is pip-installable via `pip install sphinx`.  Once installed, open a command line in the docs folder and run the following commands:
+
+```bash
+$ sudo pip install -r requirements.txt
+```
+
+This will install the requirements needed for the generating the docs. Afterwards you can run 
+
+```bash
+$ make html
+```
+ 
+The docs will be generated, the output files will be placed in the `_build/html/` directory, and can be browsed (locally) with any browser.
 
 The docs can also be found online at http://bootstrap-datepicker.readthedocs.org/.

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -22,7 +22,7 @@ import sys, os
 # The full version, including alpha/beta/rc tags.
 #release = ''
 
-
+import sphinx_rtd_theme
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
@@ -105,7 +105,8 @@ html_theme = 'sphinx_rtd_theme'
 #html_theme_options = {}
 
 # Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = ['_themes',]
+# html_theme_path = ['_themes',]
+html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,0 +1,1 @@
+sphinx_rtd_theme

